index.js

```diff
@@ -1,15 +1,13 @@
 var utils = require('loader-utils');
 
-module.exports = function (source) {
+module.exports = function(source) {
   this.cacheable();
-  var query = utils.parseQuery(this.query);
 
-  if (typeof query.search !== 'undefined' && typeof query.replace !== 'undefined') {
-    if (typeof query.flags !== 'undefined') {
-      query.search = new RegExp(query.search, query.flags);
-    }
+  var query = utils.parseQuery(this.query);
+  for (var i = 0; i < query.replace.length; i++) {
```

Comment:
Don't forget about backward compatibility for those who want replace only one option. And I personally suggest use "multiple" term instead of "replace".
```js
if (Array.isArray(query.replace) {
  ...
} else if (typeof query.search !== 'undefined' && typeof query.replace !== 'undefined') {
  ...
}
```